--- libcoyotl.old/command_line.cpp	2005-10-10 06:48:30.000000000 +0200
+++ libcoyotl/command_line.cpp	2008-07-26 22:38:01.000000000 +0200
@@ -32,6 +32,9 @@
 //-----------------------------------------------------------------------
 
 #include "command_line.h"
+
+#include <cstring>
+
 using namespace libcoyotl;
 
 command_line::command_line(int argc, char * argv[], const set<string> & bool_opts)
--- libcoyotl/sortutil.h.old	2008-07-26 22:49:47.000000000 +0200
+++ libcoyotl/sortutil.h	2008-07-26 22:49:57.000000000 +0200
@@ -38,6 +38,8 @@
 
 #include <stdexcept>
 
+#include <climits>
+
 namespace libcoyotl
 {
 
