--- libcoyotl.old/command_line.cpp	2005-10-10 06:48:30.000000000 +0200
+++ libcoyotl/command_line.cpp	2008-07-26 22:38:01.000000000 +0200
@@ -32,6 +32,9 @@
 //-----------------------------------------------------------------------
 
 #include "command_line.h"
+
+#include <cstring>
+
 using namespace libcoyotl;
 
 command_line::command_line(int argc, char * argv[], const set<string> & bool_opts)
--- libcoyotl/sortutil.h.old	2008-07-26 22:49:47.000000000 +0200
+++ libcoyotl/sortutil.h	2008-07-26 22:49:57.000000000 +0200
@@ -38,6 +38,8 @@
 
 #include <stdexcept>
 
+#include <climits>
+
 namespace libcoyotl
 {
 
--- libcoyotl-3.1.0/libcoyotl/maze_renderer.cpp.orig	2005-10-10 06:48:30.000000000 +0200
+++ libcoyotl-3.1.0/libcoyotl/maze_renderer.cpp	2013-11-08 18:00:16.823300327 +0100
@@ -36,6 +36,7 @@
 #include "maze_renderer.h"
 using namespace libcoyotl;
 
+#include <cstring>
 #include <stdexcept>
 using namespace std;
 
